docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+
+   If you don't see an email, check your spam folder.
+   If you still don't see an email, make sure your SendGrid account is fully set up (2FA enabled, email confirmed) and that your email API key is [correctly configured](#configure-email-credentials).
+
+### Configure a webhook on your machine
+
+Now it's time to configure a [webhook](/build/configure/#webhooks) on your machine to trigger the email cloud function when a person is not wearing a hard hat.
+Since you configured data to sync only when an image of a person without a hard hat is captured, configuring the webhook to trigger each time an image is synced to the cloud will produce the desired result.
+
+Configure a webhook as follows:
+
+1. Navigate to the **Config** tab of your machine.
+   Toggle the **Mode** to **Raw JSON**.
+
+2. Paste the following JSON template into your raw JSON config.
+   `"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
+
+   ```json {class="line-numbers linkable-line-numbers"}
+     "webhooks": [
+       {
+         "url": "<Insert your own cloud function URL>",
+         "event": {
+           "attributes": {
+             "data_types": ["binary"]
+           },
+           "type": "part_data_ingested"
+         }
+       }
+     ]
+   ```
+
+3. Replace the `url` value with your cloud function URL.
+   You can get this URL by copying it from the **TRIGGER** tab in the cloud function console.
+   Once you've done this, the `url` line should resemble, for example, `"url": "https://us-east1-blah-blah-123456.cloudfunctions.net/hat-email"`.
+
+4. Click **Save config**.
+
+## Test the whole system
+
+You've built all the pieces of the system and connected them together.
+Now it's time to test the whole thing.
+
+Make sure `viam-server` is running on your machine.
+Position yourself, without a hard hat, in front of your camera.
+Wait a couple of minutes for the email to arrive in your inbox, then celebrate your success when it does!
+Great work!
+
+## Next steps
+
+Here are some ways you could expand on this project:
+
+- Mount a camera on a rover and either drive it manually using remote control, or use the motion and navigation services to plan paths for the rover.
+  Or, mount cameras in multiple places.
+
+- Change your cloud function to send a different kind of notification, or trigger some other action.
+  For an example demonstrating how to configure text notifications, see the [Detect a Person and Send a Photo tutorial](/tutorials/projects/send-security-photo/).
+
+- Use a different existing model or [train your own](/ml/train-model/), to detect and send notifications about something else such as [forklifts](https://huggingface.co/keremberke/yolov8m-forklift-detection).
+
+{{< cards >}}
+{{% card link="/tutorials/projects/send-security-photo/" %}}
+{{% card link="/ml/train-model/" %}}
+{{% card link="/tutorials/configure/scuttlebot/" %}}
```

Comment:
I think that's less relevant
```suggestion
```

Reply:
I was thinking because they could put their camera on a rover? Do you think the navigation one is enough for rovers?

Reply:
I think adding too many links could be overwhelming, you could use the scutle one instead of the navigate one if you want.